website/src/components/docCarousel/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Children } from 'react';
```

Comment:
@wendyvasq May you remove this `Children` import here to cleanup the eslint warning? Once that's taken out i'm ready to approve!